frank-doc-doclet/src/main/java/org/frankframework/frankdoc/model/FrankDocModel.java

```diff
@@ -286,7 +286,11 @@ List<FrankAttribute> createAttributes(FrankClass clazz, FrankElement attributeOw
 			if(method.getParameterTypes()[0].isEnum()) {
 				log.trace("Attribute [{}] has setter that takes enum: [{}]", () -> attribute.getName(), () -> method.getParameterTypes()[0].toString());
 				attribute.setAttributeType(AttributeType.STRING);
-				attribute.setAttributeEnum(findOrCreateAttributeEnum((FrankClass) method.getParameterTypes()[0]));
+				FrankClass enumClass = (FrankClass) method.getParameterTypes()[0];
+				if(! enumClass.isPublic()) {
+					log.warn("Attribute [{}] has a setter that takes an enum that is private: [{}]", attribute, enumClass.getName());
+				}
```

Comment:
Dit blijkt uiteindelijk toch geen probleem te zijn, dus hoeft de warning ook niet.

Reply:
Heb je uitgezocht dat het geen probleem is? Ik dacht dat ik een tijdje geleden heb gezien dat het wel een probleem is voor de XSDs en niet vor de JSON, of omgekeerd. Maar ik weet het niet meer zeker.

Reply:
Ik heb gechecked dat bij attribute Adapter.msgLogLevel een private enum type hoort. Dit attribute komt alleen in de Frank!Doc als het doclet gevoed wordt met private Java elementen. Daarom moeten we https://github.com/ibissource/iaf/pull/4300 toch mergen. Als we dit doen, dan is de warning inderdaad overbodig. Ik heb hem weggehaald.

Het is belangrijk om eerst deze PR te mergen en dan pas de genoemde iaf PR. Deze PR bevat wijzigingen die ervoor zorgen dat er geen ongewenste tags worden gedefiniëerd door de output XSDs naar aanleiding van private Java classes. Ik heb gecontroleerd dat de build niet zal falen als deze volgorde wordt aangehouden.